UPGRADE-15.0.md

```diff
@@ -1167,6 +1167,10 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
     -   after these manual changes run `composer install`, `php phing standards-fix` and `php phing phpstan` commands, which will probably fail on errors caused by incompatibility strict types, if so fix them manually
     -   see #project-base-diff to update your project
 
+#### new cart items are adedd to the end of cart item list ([#3231](https://github.com/shopsys/shopsys/pull/3231))
+
+-   see #project-base-diff to update your project
+
```

Comment:
Maybe it is worth mentioning that the sort order on `Cart::$items` was changed, in case anyone have the property overridden in the project?